about.html

```diff
@@ -0,0 +1,94 @@
+  <link rel="stylesheet" href="./css/aboutsection.css">
+  
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <script src="https://kit.fontawesome.com/bdd8845187.js" crossorigin="anonymous"></script>
+  
+  <title>About MoP.</title>
+</head>
+<body>
+<main>
+  <header>
+    <nav class="navigation d-flex align-items-center justify-content-between">
+      <i class="fa-solid fa-bars open-menu"></i>
+      <a href="./index.html">MoP.™</a>
+      <ul class="navigation__list">
+        <li><a href="./index.html">Home</a></li>
+        <li><a href="index.html#program">Program</a></li>
+        <li><a href="index.html#speakers">Speakers</a></li>
+      </ul>
+    </nav>
+  </header>
+  <ul class="navigation__mobile">
+    <li><a href=""><img id="close-menu" class="close-menu" src="./images/close.png" alt="close"></a></li>
+     <li><a href="./index.html" class="navigation__mobile__links externalLink">Home</a></li>
+     <li><a href="index.html#program" class="navigation__mobile__links externalLink">Program</a></li>
+     <li><a href="index.html#speakers" class="navigation__mobile__links externalLink">Speakers</a></li>
+  </ul>
+  <section class="about">
+    <p class="about__para1 text-center">
+      "Greetings, students of Philosophy"
+    </p>
+    <h1>
+      Masters of Philosophy™ <br>
+      ANNUAL CONGRESS OF DISCOURSE <br>
+      2022 <br>
+    </h1>
+    <p class="about__para2">
+      The Congress brings together the most brilliant minds in modern Philosophy
+      to share knowledge and discuss about the latest advancements in the numerous 
+      fields of philosophy. We have philosphy professors from all over the globe 
+      who grace us with their attendance to give passionate lectures on the translations
+      of all our favorite ancient philosophers. 
+    </p>
+    <div class="p-3 text-center">
+      <h2 class="about__enq">
+        For any enquiries about the Congress of Discourse, 
+        feel free to contact us at:
+      </h2>
+      <a class="about__enq__mail" href="#">mastersofphilosophy@gmail.com</a>
+    </div>
+  </section>
+
+  <section class="statement d-flex flex-column">
+    <h2 class="text-center p-3 ml-auto mr-auto">
+      Our Statement
+    </h2>
+    <div class="statement__container">
+      <img class="rounded" src="./images/mission.jpg" alt="mission">
+      <img class="rounded" src="./images/vision.jpg" alt="vision">
+
+    </div>
+
+  </section>
+  <section class="partners">
+    <h2 class="text-center p-3 ml-auto mr-auto">
+      Partners
+    </h2>
+    <ul class="partners__list d-flex align-items-center justify-content-center">
+      <li> Plato Institute </li>
+      <li>Arksel University</li>
+      <li>Seneca Museum of Philosophy</li>
+    </ul>
+  </section>
+  <footer class="copyright d-flex justify-content-between">
```

Comment:
- [ ] [OPTIONAL] The background of the ```footer``` section in the ```About page``` should be dark, please implement this change so your design is as close as possible to the CC design 

|Your design|CC design|
|--|--|
|![Screenshot_20220526_205019](https://user-images.githubusercontent.com/25227022/170566534-14d8f20d-cc49-43c0-88dc-9959a4575c47.png)|![Screenshot_20220526_205113](https://user-images.githubusercontent.com/25227022/170566560-cab73b26-9117-4ff3-9818-f819108a1109.png)|

Reply:
- [OPTIONAL] Your Desktop Size should be divided into a top `dark bar` and a bottom `white bar`. Kindly change the background color and the text color of the Home Page footer, you have to stick to the [original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015),

- Your design:

![foobab42](https://user-images.githubusercontent.com/79058364/170608476-01608f2b-5d06-4eb4-bc4e-4209a8eb535a.png)

- [Cindy Shin](https://www.behance.net/adagio07)[Cindy Shin](https://www.behance.net/adagio07) design:

![footabou1](https://user-images.githubusercontent.com/79058364/170037105-cdec3174-540d-42d9-96a6-5390be014cae.png)